x/pairing/keeper/pairing.go

```diff
@@ -189,27 +180,34 @@ func (k Keeper) getProjectStrictestPolicy(ctx sdk.Context, project projectstypes
 	}
 
 	if !projectstypes.CheckChainIdExistsInPolicies(chainID, policies) {
-		return 0, 0, "", 0, 0, []string{}, fmt.Errorf("chain ID not found in any of the policies")
+		return projectstypes.Policy{}, "", 0, fmt.Errorf("chain ID not found in any of the policies")
 	}
 
 	geolocation := k.CalculateEffectiveGeolocationFromPolicies(policies)
 
 	providersToPair := k.CalculateEffectiveProvidersToPairFromPolicies(policies)
 	if providersToPair == uint64(math.MaxUint64) {
-		return 0, 0, "", 0, 0, []string{}, fmt.Errorf("could not calculate providersToPair value: all policies are nil")
+		return projectstypes.Policy{}, "", 0, fmt.Errorf("could not calculate providersToPair value: all policies are nil")
 	}
 
 	sub, found := k.subscriptionKeeper.GetSubscription(ctx, project.GetSubscription())
 	if !found {
-		return 0, 0, "", 0, 0, []string{}, fmt.Errorf("could not find subscription with address %s", project.GetSubscription())
+		return projectstypes.Policy{}, "", 0, fmt.Errorf("could not find subscription with address %s", project.GetSubscription())
 	}
 	allowedCU := k.CalculateEffectiveAllowedCuPerEpochFromPolicies(policies, project.GetUsedCu(), sub.GetMonthCuLeft())
 
 	projectToPair := project.Index
 
 	selectedProvidersMode, selectedProvidersList := k.CalculateEffectiveSelectedProviders(policies)
 
-	return geolocation, providersToPair, projectToPair, allowedCU, selectedProvidersMode, selectedProvidersList, nil
+	strictestPolicy := projectstypes.Policy{
+		GeolocationProfile:    geolocation,
+		MaxProvidersToPair:    providersToPair,
+		SelectedProvidersMode: selectedProvidersMode,
+		SelectedProviders:     selectedProvidersList,
+	}
+
+	return strictestPolicy, projectToPair, allowedCU, nil
```

Comment:
we dont need the projectToPair, the called already has it in the project

Reply:
done